### ISSUE TITLE
Align test doesn't work in Pop!_OS

### ISSUE THREAD

**Issue by Jakes-Lab** [hhannine/superpaper#88]
**Describe the bug**
Clicking align test doesn't put the alignment test image as the desktop background on Pop!_OS running GNOME DE.

**To Reproduce**
1. Go to the main setup page or  perspectives window
2. Click on align test
3. align test image doesn't appear

**Expected behavior**
Alignment test image appears as desktop background

**Desktop (please complete the following information):**
 - OS: Pop!_OS
 - Desktop Environment on Linux: Gnome 3.38.3
 - Superpaper package & version: PyPI superpaper 2.1.0

**Comment by hhannine** [comment]
Can you run superpaper with the --debug flag and post the output here?

**Comment by Jakes-Lab** [comment]
Sorry for the delay here is the debug output

```
/home/user/.local/lib/python3.8/site-packages
/home/user/.config/superpaper
/home/user/.config/superpaper/profiles
None
None
None
None
None
None
True
use_hotkeys: True
hk_binding_next: ('control', 'super', 'w')
hk_binding_pause: ('control', 'super', 'shift', 'p')
get_display_data output: NUM_DISPLAYS = 3, RES_ARR = [(1920, 1080), (2560, 1440), (1920, 1080)], OFF_ARR = [(0, 360), (1920, 0), (4480, 360)]
Display(resolution=(1920, 1080), digital_offset=(0, 360), phys_size_mm=(480, 270), detected_phys_size_mm=(480, 270), ppi=101.6, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='HDMI-0')
Display(resolution=(2560, 1440), digital_offset=(1920, 0), phys_size_mm=(598, 336), detected_phys_size_mm=(598, 336), ppi=108.73578595317724, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='DP-2')
Display(resolution=(1920, 1080), digital_offset=(4480, 360), phys_size_mm=(480, 270), detected_phys_size_mm=(480, 270), ppi=101.6, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='DP-1')
load_system: archive_file not found: /home/user/.config/superpaper/display_systems.dat
START Listing profiles for menu.
hkBinding: ('control', 'super', 'shift', 'h')
Listed profile: example
hkBinding: ('control', 'super', 'shift', 'g')
Listed profile: example_multi
END Listing profiles for menu.
No previous profile was found.
hkBinding: ('control', 'super', 'shift', 'h')
Listed profile: example
hkBinding: ('control', 'super', 'shift', 'g')
Listed profile: example_multi
Registering binding:                                 ('control', 'super', 'shift', 'h') for profile: example
Registering binding:                                 ('control', 'super', 'shift', 'g') for profile: example_multi
use_hotkeys: True
hk_binding_next: ('control', 'super', 'w')
hk_binding_pause: ('control', 'super', 'shift', 'p')
get_display_data output: NUM_DISPLAYS = 3, RES_ARR = [(1920, 1080), (2560, 1440), (1920, 1080)], OFF_ARR = [(0, 360), (1920, 0), (4480, 360)]
Display(resolution=(1920, 1080), digital_offset=(0, 360), phys_size_mm=(480, 270), detected_phys_size_mm=(480, 270), ppi=101.6, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='HDMI-0')
Display(resolution=(2560, 1440), digital_offset=(1920, 0), phys_size_mm=(598, 336), detected_phys_size_mm=(598, 336), ppi=108.73578595317724, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='DP-2')
Display(resolution=(1920, 1080), digital_offset=(4480, 360), phys_size_mm=(480, 270), detected_phys_size_mm=(480, 270), ppi=101.6, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='DP-1')
load_system: archive_file not found: /home/user/.config/superpaper/display_systems.dat
hkBinding: ('control', 'super', 'shift', 'h')
Listed profile: example
hkBinding: ('control', 'super', 'shift', 'g')
Listed profile: example_multi
use_hotkeys: True
hk_binding_next: ('control', 'super', 'w')
hk_binding_pause: ('control', 'super', 'shift', 'p')

```

**Comment by hhannine** [comment]
Did it cut off? I'm looking for a specific thing that's not in the log you posted. Try to use the align test while Superpaper is running with the debug on.

Have you tried to set a wallpaper with Superpaper? Does that work?

**Comment by Tom-ST** [comment]
It also doesn't seem to work on Windows 10.
I started superpaper, loaded a setting profile, applied settings, clicked align and then opened the log.

Log:

> Enabled logging to file.
> use_hotkeys: True
> hk_binding_next: ('control', 'super', 'w')
> hk_binding_pause: ('control', 'super', 'shift', 'p')
> get_display_data output: NUM_DISPLAYS = 2, RES_ARR = [(1080, 1920), (1920, 1080)], OFF_ARR = [(0, 0), (1080, 147)]
> Display(resolution=(1080, 1920), digital_offset=(0, 0), phys_size_mm=(270, 480), detected_phys_size_mm=(270, 480), ppi=101.6, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='\\\\.\\DISPLAY2')
> Display(resolution=(1920, 1080), digital_offset=(1080, 147), phys_size_mm=(510, 287), detected_phys_size_mm=(510, 287), ppi=95.6235294117647, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='\\\\.\\DISPLAY1')
> load_system: archive_file not found: C:\Users\Tom\Desktop\superpaper-portable\display_systems.dat
> Listed profile: Test
> hkBinding: ('control', 'super', 'shift', 'h')
> Listed profile: example
> hkBinding: ('control', 'super', 'shift', 'g')
> Listed profile: example_multi
> bezels_to_bitmap: bez_rects: ((0, 0), (0, 0))
> bezels_to_bitmap: bez_rects: ((0, 0), (0, 0))
> bezels_to_bitmap: bez_rects: ((0, 0), (0, 0))
> bezels_to_bitmap: bez_rects: ((0, 0), (0, 0))
> Test
> advanced
> False
> None
> None
> None
> None
> ['C:\\Users\\Tom\\Desktop\\2109.jpg']
> perspective preset: default
> Listed profile: Test
> hkBinding: ('control', 'super', 'shift', 'h')
> Listed profile: example
> hkBinding: ('control', 'super', 'shift', 'g')
> Listed profile: example_multi
> perspective preset: default
> Listed profile: Test
> hkBinding: ('control', 'super', 'shift', 'h')
> Listed profile: example
> hkBinding: ('control', 'super', 'shift', 'g')
> Listed profile: example_multi
> Unreg hotkey ('control', 'super', 'shift', 'g')
> Unreg hotkey ('control', 'super', 'w')
> Unreg hotkey ('control', 'super', 'shift', 'h')
> Unreg hotkey ('control', 'super', 'shift', 'p')
> perspective preset: default
> Listed profile: Test
> hkBinding: ('control', 'super', 'shift', 'h')
> Listed profile: example
> hkBinding: ('control', 'super', 'shift', 'g')
> Listed profile: example_multi
> Registering binding:                                 None for profile: Test
> Registering binding:                                 ('control', 'super', 'shift', 'h') for profile: example
> Registering binding:                                 ('control', 'super', 'shift', 'g') for profile: example_multi
> perspective preset: default
> bezels_to_bitmap: bez_rects: ((0, 0), (0, 0))
> bezels_to_bitmap: bez_rects: ((0, 0), (0, 0))
> onApply profile: saved C:\Users\Tom\Desktop\superpaper-portable\profiles\Test.profile
> perspective preset: default
> perspective preset: default
> Listed profile: Test
> hkBinding: ('control', 'super', 'shift', 'h')
> Listed profile: example
> hkBinding: ('control', 'super', 'shift', 'g')
> Listed profile: example_multi
> Start profile: Test
> Check if the starting profile is already running: Test
> name check: Test, Test
> Running quick profile job with profile: Test
> quickswitch file lookup: ['Test-b.jpg']
> Starting timed profile job with profile: Test
> get_display_data output: NUM_DISPLAYS = 2, RES_ARR = [(1080, 1920), (1920, 1080)], OFF_ARR = [(0, 0), (1080, 147)]
> Display(resolution=(1080, 1920), digital_offset=(0, 0), phys_size_mm=(270, 480), detected_phys_size_mm=(270, 480), ppi=101.6, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='\\\\.\\DISPLAY2')
> Display(resolution=(1920, 1080), digital_offset=(1080, 147), phys_size_mm=(510, 287), detected_phys_size_mm=(510, 287), ppi=95.6235294117647, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='\\\\.\\DISPLAY1')
> load_system: archive_file not found: C:\Users\Tom\Desktop\superpaper-portable\display_systems.dat
> running profile job with profile: Test
> Running a one-off wallpaper change.
> ['C:\\Users\\Tom\\Desktop\\2109.jpg']
> get_ppi_norm_offsets: [(0, 0), (1080, 386)]
> get_ppi_norm_crops: [(0, 0, 1080, 1920), (1080, 386, 3120, 1534)]
> G_A_DSYS.use_perspective: True, prof.perspective: default
> Wrote active profile: Test
> Canvas size: [3000, 1920]
> Uncaught exception type: <class 'system_hotkey.system_hotkey.SystemRegisterError'>
> NoneType: None
> Exception: The bind could be in use elsewhere: h
> NoneType: None
> <traceback object at 0x000001AB5235B2C8>
> NoneType: None
> get_display_data output: NUM_DISPLAYS = 2, RES_ARR = [(1080, 1920), (1920, 1080)], OFF_ARR = [(0, 0), (1080, 147)]
> Display(resolution=(1080, 1920), digital_offset=(0, 0), phys_size_mm=(270, 480), detected_phys_size_mm=(270, 480), ppi=101.6, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='\\\\.\\DISPLAY2')
> Display(resolution=(1920, 1080), digital_offset=(1080, 147), phys_size_mm=(510, 287), detected_phys_size_mm=(510, 287), ppi=95.6235294117647, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='\\\\.\\DISPLAY1')
> load_system: archive_file not found: C:\Users\Tom\Desktop\superpaper-portable\display_systems.dat
> Computed PPIs: [101.51645945079716, 95.77857261227383]
> relative pixel densities: [1.0, 0.9434782608695652]
> ['C:\\Users\\Tom\\Desktop\\superpaper-portable\\superpaper\\resources\\test.png']
> get_ppi_norm_offsets: [(0, 0), (1080, 386)]
> get_ppi_norm_crops: [(0, 0, 1080, 1920), (1080, 386, 3120, 1534)]
> G_A_DSYS.use_perspective: True, prof.perspective: default
> super', 'shift', 'g')
> Listed profile: example_multi
> Start profile: Test
> Check if the starting profile is already running: Test
> name check: Test, Test
> Running quick profile job with profile: Test
> quickswitch file lookup: ['Test-b.jpg']
> Starting timed profile job with profile: Test
> get_display_data output: NUM_DISPLAYS = 2, RES_ARR = [(1080, 1920), (1920, 1080)], OFF_ARR = [(0, 0), (1080, 147)]
> Display(resolution=(1080, 1920), digital_offset=(0, 0), phys_size_mm=(270, 480), detected_phys_size_mm=(270, 480), ppi=101.6, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='\\\\.\\DISPLAY2')
> Display(resolution=(1920, 1080), digital_offset=(1080, 147), phys_size_mm=(510, 287), detected_phys_size_mm=(510, 287), ppi=95.6235294117647, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='\\\\.\\DISPLAY1')
> load_system: archive_file not found: C:\Users\Tom\Desktop\superpaper-portable\display_systems.dat
> running profile job with profile: Test
> Running a one-off wallpaper change.
> ['C:\\Users\\Tom\\Desktop\\2109.jpg']
> get_ppi_norm_offsets: [(0, 0), (1080, 386)]
> get_ppi_norm_crops: [(0, 0, 1080, 1920), (1080, 386, 3120, 1534)]
> G_A_DSYS.use_perspective: True, prof.perspective: default
> Wrote active profile: Test
> Canvas size: [3000, 1920]
> Uncaught exception type: <class 'system_hotkey.system_hotkey.SystemRegisterError'>
> NoneType: None
> Exception: The bind could be in use elsewhere: h
> NoneType: None
> <traceback object at 0x000001AB5235B2C8>
> NoneType: None
> get_display_data output: NUM_DISPLAYS = 2, RES_ARR = [(1080, 1920), (1920, 1080)], OFF_ARR = [(0, 0), (1080, 147)]
> Display(resolution=(1080, 1920), digital_offset=(0, 0), phys_size_mm=(270, 480), detected_phys_size_mm=(270, 480), ppi=101.6, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='\\\\.\\DISPLAY2')
> Display(resolution=(1920, 1080), digital_offset=(1080, 147), phys_size_mm=(510, 287), detected_phys_size_mm=(510, 287), ppi=95.6235294117647, ppi_norm_resolution=None, ppi_norm_offset=None, ppi_norm_bezels=(0, 0), perspective_angles=(0, 0), name='\\\\.\\DISPLAY1')
> load_system: archive_file not found: C:\Users\Tom\Desktop\superpaper-portable\display_systems.dat
> Computed PPIs: [101.51645945079716, 95.77857261227383]
> relative pixel densities: [1.0, 0.9434782608695652]
> ['C:\\Users\\Tom\\Desktop\\superpaper-portable\\superpaper\\resources\\test.png']
> get_ppi_norm_offsets: [(0, 0), (1080, 386)]
> get_ppi_norm_crops: [(0, 0, 1080, 1920), (1080, 386, 3120, 1534)]
> G_A_DSYS.use_perspective: True, prof.perspective: default
> 
>

**Comment by hhannine** [comment]
This has been resolved: 286cfc37f260a286306c991e69abdacf77a3a88e
